src/stories/simple-vertical-list/index.stories.tsx

```diff
@@ -1,15 +1,17 @@
+
```

Comment:
I think we should create a dedicated story for knobs 🙂

Reply:
Your call :)